scripts/qa_crowbarsetup.sh

```diff
@@ -5028,6 +5028,21 @@ function onadmin_run_cct
             done
         fi
 
+        # NOTE(aplanas) This workaround for bsc#1024742 will inject
+        # overrides to compute nodes the belong to the pacemaker
+        # remote cluster. This override will decrease the
+        # TimeoutStopSec from 30min to 30secs, making the boot process
+        # faster.
+        if iscloudver 7plus; then
+            local remotes=$(crowbarctl proposal show \
+                --filter deployment.pacemaker.elements.pacemaker-remote \
+                --plain pacemaker services | cut -d' ' -f2)
+            local override=/etc/systemd/system/pacemaker_remote.d/override.conf
+            for node in $remotes; do
+                ssh $node "echo TimeoutStopSec=30s > $override; systemctl daemon-reload"
```

Comment:
According to documentation when you change a service or override, you need to rescan with this command